LambdaEngine/Include/Memory/API/Malloc.h

```diff
@@ -6,8 +6,7 @@
 * Delete and Release
 */
 
-#define DELETE_OBJECT(object)	delete(object); (object) = nullptr
-#define SAFEDELETE(object)		if ((object))	{ DELETE_OBJECT(object); }
+#define SAFEDELETE(object)		delete(object); (object) = nullptr
```

Comment:
Did we not lose the safety feature if we delete the object without checking?

Reply:
Calling delete on `nullptr` does nothing. It was an unnecessary check.